xc/redis_pool.lua

```diff
@@ -31,7 +32,7 @@ function _M.acquire()
 
   conn:set_timeout(_M.timeout)
 
-  local ok, err = conn:connect(_M.host, _M.port)
+  local ok, err = conn:connect(threescale_utils.resolve(_M.host, _M.port))
```

Comment:
I implemented some error handling and tested it. Check the 2 new commits that I added.